sips/sip-028/sip-028-sbtc_peg.md

```diff
@@ -0,0 +1,203 @@
+
+#### For Stackers:
+
+In order for this SIP to activate, the following criteria must be met by the set of Stacked STX:
+
+- At least 80 million Stacked STX must vote, with least 66% (48 million) voting "yes".
+
+The voting addresses will be:
+
+| **Vote** | **Bitcoin Address**              | **Stacks Address**                    | Message      | ASCII-encoded message                      | Bitcoin script                                                                                  |
+| -------- | -------------------------------- | ------------------------------------- | ------------ | ------------------------------------------ | ----------------------------------------------------------------------------------------------- |
+| yes      | `11111111111mdWK2VXcrA1e7dnvidC` | `SP00000000001WPAWSDEDMQ0B9J72P0KAK2` | `yes-sip-28` | `000000000000000000007965732d7369702d3238` | `OP_DUP` `OP_HASH160` `000000000000000000007965732d7369702d3238` `OP_EQUALVERIFY` `OP_CHECKSIG` |
+| no       | `111111111111ACW5wa4RwyeKYEAzMD` | `SP000000000006WVSDEDMQ0B9J73E2TN78`  | `no-sip-28`  | `00000000000000000000006e6f2d7369702d3238` | `OP_DUP` `OP_HASH160` `00000000000000000000006e6f2d7369702d3238` `OP_EQUALVERIFY` `OP_CHECKSIG` |
+
+The addresses have been generated as follows:
+
+- Encode `<message>` in ASCII, with 0-padding.
+- Use the resulting `<encoding>` in the Bitcoin script`OP_DUP` `OP_HASH160` `<encoding>` `OP_EQUALVERIFY` `OP_CHECKSIG`.
+- The Bitcoin address is the `base58check` of the hash of the Bitcoin script above.
+- The Stacks address is the `c32check-encoded` Bitcoin address.
+
+Stackers (pool and solo) vote by sending Stacks dust to the corresponding Stacks address from the account where their Stacks are locked.
+
+Solo stackers only can also vote by sending a bitcoin dust transaction (6000 sats) to the corresponding bitcoin address.
+
+#### For Non-Stackers:
+
+Users with liquid STX can vote on proposals using the Ecosystem DAO. Liquid STX is the user’s balance, less any STX they have locked in the PoX stacking protocol, at the block height at which the voting started (preventing the same STX from being transferred between accounts and used to effectively double vote). This is referred to generally as "snapshot" voting.
+
+For this SIP to pass, 66% of all liquid STX committed by voting must be in favor of the proposal. This precedent was set by [SIP-015](https://github.com/stacksgov/sips/blob/main/sips/sip-015/sip-015-network-upgrade.md).
+
+The act of not voting is the act of siding with the outcome, whatever it may be. We believe that these thresholds are sufficient to demonstrate interest from Stackers -- Stacks users who have a long-term interest in the Stacks blockchain's successful operation -- in performing this upgrade.
+
+## Appendix
+[1] https://github.com/stacks-network/sbtc/issues/52
+
+[2] https://github.com/stacks-network/sbtc/pull/186
+
+### Specification
```

Comment:
consider moving this to main Specification section, or renaming this to appropriate section name that could be referenced in main Specification

Reply:
Not sure i follow the thinking here - structurally, this specification is part of the Appendix. 
are you asking to either move this section to the top level specification, or reference this appendix from the top-level?

@andrerserrano @aldur

Reply:
either way is fine - moving to the top or if it better fits in appendix, then may be include references within top level specification

Reply:
i think i see what you're getting at - so in the case where this appendix specification remains where it is, you'd like to see the top-level specification annotate to the bottom-level specification?

Reply:
@jiga We intentionally did not include this in the Spec because how deposits and withdrawals work are outside of the scope for this SIP. I added a note in the Spec section for the reader find more detail in the appendix.  https://github.com/stacksgov/sips/pull/186/commits/3aece3aad3fab822bf7ef7056a07990f02b73c05

Reply:
thanks @andrerserrano this looks good now